src/index.ts

```diff
@@ -242,10 +243,45 @@ async function wranglerCommands() {
 				}
 			}
 
-			await exec(`${packageManager.exec} wrangler ${command}`, args, {
+			// Used for saving the wrangler output
+			let stdOut = "";
+			let stdErr = "";
+
+			// Construct the options for the exec command
+			const options = {
 				cwd: config["workingDirectory"],
 				silent: config["QUIET_MODE"],
-			});
+				listeners: {
+					stdout: (data: Buffer) => {
+						stdOut += data.toString();
+					},
+					stderr: (data: Buffer) => {
+						stdErr += data.toString();
+					},
+				},
+			};
+
+			// Execute the wrangler command
+			await exec(`${packageManager.exec} wrangler ${command}`, args, options);
+
+			// Set the outputs for the command
+			setOutput("command-output", stdOut);
+			setOutput("command-stderr", stdErr);
+
+			// Check if this command is a workers or pages deployment
+			if (
+				command.startsWith("deploy") ||
+				command.startsWith("publish") ||
+				command.startsWith("pages publish")
+			) {
+				// If this is a workers or pages deployment, try to extract the deployment URL
+				let deploymentUrl = "";
+				const deploymentUrlMatch = stdOut.match(/https?:\/\/[a-zA-Z0-9-\.\/]+/);
```

Comment:
while what you have still works, it's not necessary to escape the `.` and `/` characters inside of character class expressions.

Reply:
Perhaps I'm not familiar enough with regular expressions in JavaScript but I would think that removing the escape characters would cause all sorts of errors here. 

I got a fair amount of errors when trying to test this out so perhaps we should leave it the way that it is for now since it is working as expected?

Reference:

![image](https://github.com/cloudflare/wrangler-action/assets/23362539/0f153411-1c6b-4aa9-a6e2-9a02242fc3c1)

Reply:
Sorry for the confusion. To be clear, I’m referring only to the `\.\/` _within the character class expression_ (the stuff between `[]`), not the `\/\/` outside of it, which do still need to be escaped. 
```suggestion
				const deploymentUrlMatch = stdOut.match(/https?:\/\/[a-zA-Z0-9-./]+/);
```

Reply:
Oh interesting! I went ahead and committed your suggestion